sdks/java/io/jms/pom.xml

```diff
@@ -59,6 +46,15 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
I think this is fine too, but I'd prefer that we have one way of suppressing false positives. Right now, all suppressions seem to be in `findbugs-filter.xml`, whereas this would introduce another way of suppressing some warnings.

I'd recommend to move the suppressions there, if possible.